package.json

```diff
@@ -2,11 +2,10 @@
   "name": "poolifier",
   "version": "1.2.1",
   "description": "Library on top of node js worker threads that implement various worker pools type",
-  "main": "index.js",
-  "types": "index.d.ts",
+  "main": "lib/index.js",
   "scripts": {
-    "build": "npm install",
-    "test": "standard && nyc mocha --exit --timeout 20000 tests/*test.js ",
+    "build": "npm install && rimraf lib && tsc",
```

Comment:
Integrating grunt for the build might be not difficult and will simplify scripts.

Reply:
We could do this (e.g. using [rollup](https://github.com/rollup/rollup)), but I think it's not really needed  
I personally prefer to stick to native tools if it's that easy like currently

Reply:
rollup, grunt, etc. and similar lightweight bundling tools (not like webpack ;)) have more advantages than drawbacks: build time, build size, build type, hooks handling, ...
Postponing its integration later on is possible but since it will be done at some point ...

Reply:
That's a good idea, you should create a new issue for that task  
I would prefer rollup, because I heard gulp and grunt are dead/outdated tools and rollup is the way to go for library bundling (https://www.npmtrends.com/rollup-vs-grunt-vs-gulp)

<img src="https://user-images.githubusercontent.com/7195563/107117828-3ab3e180-687d-11eb-96da-ab4a0fa71849.png" width="400px"/>


I as a frontend developer just use `vue-cli` ^^ and for libraries like my prettier plugin I just use `tsc`
So that's why I don't really have any knowledge about lib-bundlers

Reply:
In the meantime, could you please remove `npm install` from the build script if you do not plan to merge one of the PR I've made?
GitHub CI is then going to do it twice and build scripts always assume deps are installed.